internal/controller/server/server.go

```diff
@@ -3,23 +3,27 @@ package server
 import (
 	"net/http"
 	"remote/internal/controller/server/handlers"
+	"remote/internal/model"
 
 	"github.com/gorilla/mux"
 )
 
-type server struct {
+type Server struct {
```

Comment:
What is the reason for making this struct public?

Reply:
No real reason, changed back to private